src/WMOToast/WMOToastRedux.js

```diff
@@ -0,0 +1,103 @@
+import Immutable from 'seamless-immutable';
+
+import { stringArrayToObject } from '../redux/reduxUtils';
+
+/* ------------- Toast actions ------------- */
+
+export const actions = stringArrayToObject(['DISPLAY_ERROR', 'DISPLAY_WARNING', 'DISPLAY_INFO', 'HIDE']);
+
+export const actionCreators = {
+  hide() {
+    return {
+      type: actions.HIDE
+    };
+  },
+  displayError(message, duration = 4000) {
```

Comment:
I'd use a common aux function for `displayError`, `displayWarning` and `displayInfo`.